specificatie/filter.yaml

```diff
@@ -5,23 +5,14 @@ info:
   contact: {}
 paths: {}
 components:
-  parameters:
-    Fields:
-      name: fields
-      in: query
-      required: true
-      schema:
-        $ref: '#/components/schemas/Fields'
-
   schemas:
-    Fields:
+    Field:
       description: |
-          Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een door komma's gescheiden lijst van property namen op te geven.
-          Bij opgave van niet-bestaande properties wordt een 400 Bad Request teruggegeven. Wanneer de fields parameter niet is opgegeven, worden alle properties met een waarde teruggegeven.
+          Bij opgave van niet-bestaande properties wordt een 400 Bad Request teruggegeven.
```

Comment:
properties is volgens mij geen goede term hier. In wezen is het geen property maar de pad naar een property. Zelf zit ik te twijfelen tussen `property path` of `pad van een veld`

Reply:
Volgens mij gebruiken we overal de term property dus ik denk niet dat we het hier ineens over velden moeten gaan hebben. Ik zou hier dus kiezen voor 'property paths'.